activesupport/lib/active_support/core_ext/string/output_safety.rb

```diff
@@ -16,7 +16,9 @@ module Util
                                 "\u{200C}-\u{200D}\u{2070}-\u{218F}\u{2C00}-\u{2FEF}\u{3001}-\u{D7FF}\u{F900}-\u{FDCF}" \
                                 "\u{FDF0}-\u{FFFD}\u{10000}-\u{EFFFF}"
     TAG_NAME_START_REGEXP = /[^#{TAG_NAME_START_REGEXP_SET}]/
-    TAG_NAME_FOLLOWING_REGEXP = /[^#{TAG_NAME_START_REGEXP_SET}\-.0-9\u{B7}\u{0300}-\u{036F}\u{203F}-\u{2040}]/
+    TAG_NAME_FOLLOWING_REGEXP_SET = "#{TAG_NAME_START_REGEXP_SET}\\-.0-9\u{B7}\u{0300}-\u{036F}\u{203F}-\u{2040}"
+    TAG_NAME_FOLLOWING_REGEXP = /[^#{TAG_NAME_FOLLOWING_REGEXP_SET}]/
```

Comment:
Although this follows the naming pattern set above in `TAG_NAME_START_REGEXP`, I believe both are poor names. The names describe regexes that _do not match_ characters from the related ranges ranges (`TAG_NAME_START_REGEXP_SET` and `TAG_NAME_FOLLOWING_REGEXP_SET`). The name may lead readers to the mistaken belief that they _match_ characters from those sets.

Should we name both constants to something more correct, like `CODEPOINTS_NOT_IN_TAG_NAME_START` or `INVALID_TAG_NAME_START_CODEPOINTS`? It looks like these constants are only used in this file.

Reply:
👍 I like `INVALID_TAG_NAME_START_CODEPOINTS`

Reply:
Great idea!

I updated the regex names to reflect that they're matching _invalid_ characters, and squashed into https://github.com/rails/rails/commit/d5695001f1a7316ed1bb7886c1ff0b6aea95c944